english/find-exit/into-forest/forest.md

```diff
@@ -17,4 +17,16 @@ Version control rules
 
 Managed projects is the aim
 
-Peace of mind the gain.
\ No newline at end of file
+Peace of mind the gain.
+
+But only for a short while
+
+That something watching you emerges from the shadows
+
+It's a cougar, stalking it's prey.  It approaches you 
+
+You can do nothing but wait for it to make it's move
+
+Then it asks "What's your name?"
+
+[Do you start a conversation with the cougar?] (https://github.com/zmorris1/create-your-own-adventure/blob/talk-to-cougar/english/find-exit/into-forest/talk-to-cougar/talk-to-cougar.md)
```

Comment:
It looks like this line is the reason Casey still won't merge your pull request (since it's too long). You should also be using a relative link rather than an absolute link (just `talk-to-cougar/talk-to-cougar.md`), which will make the line shorter.